setup.py

```diff
@@ -17,7 +17,7 @@
 URL = "https://github.com/OFFIS-DAI/mango"
 EMAIL = "mango@offis.de"
 AUTHOR = "mango Team"
-REQUIRES_PYTHON = ">=3.7.0"
+REQUIRES_PYTHON = ">=3.9.0"
```

Comment:
Why do we need 3.9?

Reply:
I generally dont think we need to be too strict with increasing minimum python version, However, I also feel increasing it should have a significant benefit.